sources/packages/api/src/route-controllers/institution/institution.aest.controller.ts

```diff
@@ -64,4 +79,96 @@ export class InstitutionAESTController extends BaseController {
     }
     await this.institutionService.updateInstitution(institutionId, payload);
   }
+
+  /**
+   * Controller method to get all institution users with the
+   * given institutionId.
+   * @queryParm page, page number if nothing is passed then
+   * DEFAULT_PAGE_NUMBER is taken
+   * @queryParm pageLimit, page size or records per page, if nothing is
+   * passed then DEFAULT_PAGE_LIMIT is taken
+   * @queryParm searchName, user's name keyword to be searched
+   * @queryParm sortField, field to be sorted
+   * @queryParm sortOrder, order to be sorted
+   * @returns All the institution users for the given institution
+   * with total count.
+   */
+  @Get(":institutionId/user")
+  async getInstitutionUsers(
+    @Param("institutionId") institutionId: number,
+    @Query(PaginationParams.SearchCriteria) searchName: string,
+    @Query(PaginationParams.SortField) sortField: string,
+    @Query(PaginationParams.SortOrder) sortOrder = FieldSortOrder.ASC,
+    @Query(PaginationParams.Page) page = DEFAULT_PAGE_NUMBER,
+    @Query(PaginationParams.PageLimit) pageLimit = DEFAULT_PAGE_LIMIT,
+  ): Promise<PaginatedResults<InstitutionUserAPIOutDTO>> {
+    return this.institutionControllerService.getInstitutionUsers(
+      institutionId,
+      {
+        page: page,
```

Comment:
This one can also be simplified, not a blocker.

Reply:
I am changing.